### DISCUSSION TITLE
How is GHG for US Consumption calculated in the import factors report?

### DISCUSSION THREAD

**Discussion by cclin130** [USEPA/USEEIO#44] [Q&A]
I'd like to reproduce the values in this chart with new import factors I have created:

<img width="579" alt="image" src="https://github.com/user-attachments/assets/4e277b20-89b3-4085-b95c-40a6b36dee5a">

Two questions:

1. is `calculateEEIOModel` in the `useeior` repo the right helper function for this?
2. is this calculation using the final or direct perspective?

Thanks in advance!

**Comment by cclin130** [comment]
Ah actually, I noticed for my purposes that @bl-young has already given me a helper function to do this in a shared script!

**Reply by bl-young** [reply]
Great - also noting that the total result will be the same no matter the perspective.

**Reply by cclin130** [reply]
makes sense! that was my hunch but wanted to confirm :)
